### PR TITLE
Make BSD-ness more prominent

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Broad Institute, Inc.
```
Comment (mcovarr) [inline LICENSE.txt]: can we say 2016 now or does that need to wait 3 weeks?

**Comment by mcovarr** [comment]
:+1:

File: LICENSE.txt (hunk 1 above)
Comment (geoffjentry) [inline LICENSE.txt]: Don't knock my copypasta

**Comment by Horneth** [comment]
:+1:

**Pull request merged**